tests/unit/test_result.py

```diff
@@ -20,7 +20,7 @@ def create_fake_pkg(tmpdir):
 @pytest.fixture()
 def no_hostname_envvar():
     hostname = os.getenv("HOSTNAME")
```

Comment:
perhaps a better alternative is to use pytest's builtin `monkeypatch.setenv` ?

Reply:
Right, I forgot it exists. I can get to it tomorrow, I'm already away from keyboard.

Reply:
I've rebased the PR to use it, thanks for the suggestion.